### PR TITLE
Feature userinfo

### PR DESCRIPTION
**Pull request by olvrb**
# Changes
* Add userinfo command.

### CODE REVIEW

**Comment by jacc** [comment]
Might be hard, but total message count might be useful in this. Just so that people on mobile can get an accurate show at how many messages they have.

**Comment by olvrb** [comment]
Not possible with a bot account.

**Comment by jacc** [comment]
Alright.

**Comment by nullpixel** [comment]
Message count will be a possibility when the database stuff is done, as we're storing all messages in the database so we can just run a `.count().where()`

File: src/discord/commands/utilities/userinfo.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        ]
+    },
+    handler: async (message, next) => {
+        // Define constants for ease of accesss.
+        const client = message.client;
+        const guild = message.guild;
+
+        //TODO: change type from any to User.
+        // Simple hack to avoid TypeErrors
+        let [_user]: any | User = message.args;
```
Comment (nullpixel) [inline src/discord/commands/utilities/userinfo.ts:29]: cc/ @EricRabil can we get a nicer way to do this?

**Review by nullpixel — APPROVED** [review]
Core code is fine, would be nice to resolve the BotKit type stuff before merging but I understand that this is not your fault.

File: src/discord/commands/utilities/userinfo.ts (hunk 1 above)
Comment (olvrb) [reply src/discord/commands/utilities/userinfo.ts:29]: Definitely.

**Pull request merged**